tests/terraform/checks/a example skip/main.tf

```diff
@@ -0,0 +1,27 @@
+
+resource "azurerm_storage_account" "default" {
+  #checkov:skip=CKV_AZURE_33
+  name                     = "storageaccountname"
+  resource_group_name      = "azurerm_resource_group.example.name"
+  location                 = "azurerm_resource_group.example.location"
+  account_tier             = "Standard"
+  account_replication_type = "GRS"
+}
+
+resource "azurerm_storage_account" "skip_more_than_one" {
+  #checkov:skip=CKV_AZURE_33,CKV_AZURE_59: Skipped by user
+  name                     = "storageaccountname"
+  resource_group_name      = "azurerm_resource_group.example.name"
+  location                 = "azurerm_resource_group.example.location"
+  account_tier             = "Standard"
+  account_replication_type = "GRS"
+}
+
+resource "azurerm_storage_account" "skip_invalid" {
+  #checkov:skip=CKV_AZURE_33,bla bla bla: Skipped by user
+  name                     = "storageaccountname"
+  resource_group_name      = "azurerm_resource_group.example.name"
+  location                 = "azurerm_resource_group.example.location"
+  account_tier             = "Standard"
+  account_replication_type = "GRS"
+}
```

Comment:
It would be great if you could create another resource that we are skipping multiple checks (maybe all the checks that the resource should fail on).